nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentManagerImpl.java

```diff
@@ -474,13 +489,12 @@ public Comment createComment(CoreSession session, Comment comment) throws Illega
         }
         DocumentModel docToComment = session.getDocument(commentRef);
         DocumentModel commentModel = session.createDocumentModel(COMMENT_DOC_TYPE);
+        commentModel.setPropertyValue("dc:created", Calendar.getInstance());
```

Comment:
Why do we need this ? Should be done by `DublinCoreService`

Reply:
We have to set it manually because this value is overriden when using the `CommentConverter`